global/wizard.sh

```diff
@@ -1,26 +1,26 @@
 #!/bin/bash
 DIR=$(pwd)
 CLANG=${CLANG:-C.UTF-8}
-IMAGE=${IMAGE:-netways/showoff:0.19.6}
+IMAGE=${IMAGE:-netways/showoff:0.20.4}
 CNAME=${CNAME:-showoff}
 TRAINING=${TRAINING:-$(basename "$DIR")}
-DOCKER=${DOCKER:-$(command -v docker)}
+RUNTIME=${RUNTIME:-$(command -v docker)}
 GIT=${GIT:-$(command -v git)}
 SCRIPT_DIR=$(cd $(dirname $0); pwd -P)
 NO_RESET=${NO_RESET:-""}
 
-# Functions
-
+# Run given command in the showoff container
 execdocker () {
-  if [[ -n $($DOCKER ps -aq -f name=$CNAME 2> /dev/null) ]]; then
-    $DOCKER rm -f $CNAME 2> /dev/null
+  if [[ -n $($RUNTIME ps -aq -f name=$CNAME 2> /dev/null) ]]; then
+    $RUNTIME rm -f $CNAME 2> /dev/null
   fi
 
-  $DOCKER run \
+  $RUNTIME run \
     -it \
     --name=$CNAME \
     --rm \
     -p 9090:9090 \
+    --user $(id -u) \
```

Comment:
Requires also `--userns=keep-id` if using podman otherwise it will fail.

Reply:
Thanks for the hint. Fixed it.